apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md

```diff
@@ -0,0 +1,68 @@
+---
+eleventyNavigation:
+    key: 'Numeric Stepper - Form'
+    parent: numeric-stepper
+---
+
+## Overview
+The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, maintaining a consistent form field appearance in form interfaces. This allows users to easily adjust and enter numerical data with ease.
+
+Numeric steppers [form] resemble the structure of classic form fields, but allow for precise value entry through the integrated text input while also offering the convenience of incremental adjustments using the provided buttons.
```

Comment:
This sentence is a bit wordy, in my opinion, we should be aiming for simpler words and phrases that are easy for anyone to understand on a Friday afternoon 😄

Reply:
I agree, what about something like:

`Numeric steppers [form] allow a user to enter precise values, either by inputting text or by using the increment or decrement buttons.`

Reply:
Yeah that's great!